hearpreprocess/dcase2016_task2.py

```diff
@@ -72,6 +73,7 @@
                     "md5": "14539d85dec03cb7ac75eb62dd1dd21e",
                 },
             ],
+            "max_task_duration_by_split": {"train": None, "valid": None, "test": None},
```

Comment:
Are we asserting that max_task_duration_by_split must exist for every "mode" if "split_mode" == trainvaltest? Same for split_mode == ...kfold and max_task_duration_by_fold must exist? If not, we should create a separate PR immediately to sanity check this.

Reply:
It is there in the config validator